Tools/TestWebKitAPI/TestBundle/TestWebKitAPI.xctestplan

```diff
@@ -14,7 +14,7 @@
   "testTargets" : [
     {
       "target" : {
-        "containerPath" : "container:TestWebKitAPI.xcodeproj",
+        "containerPath" : "container:..\/OpenSource\/Tools\/TestWebKitAPI\/TestWebKitAPI.xcodeproj",
```

Comment:
This doesn't seem right. Nothing guarantees someone's WebKit checkout is in a folder named OpenSource.